internal/controllers/machinedeployment/machinedeployment_rollout_rollingupdate_test.go

```diff
@@ -925,6 +1476,152 @@ func Test_RollingUpdateSequences(t *testing.T) {
 			},
 			desiredMachineNames: []string{"m10", "m11", "m12", "m13", "m14", "m15"}, // NOTE: Machines created before the spec change are deleted
 		},
+
+		// Rollout with In-place updates
+
+		{ // scale out by 1 (maxSurge 0)
```

Comment:
```suggestion
		{ // scale out by 1
```
(we did the same when introducing the regular update cases, this info was wrong)